docs/content/docs/tutorials/memorydb-example.md

```diff
@@ -0,0 +1,288 @@
+---
+title: "Deploy MemoryDB Cluster Using the ACK MemoryDB Controller"
+description: "Create managed MemoryDB Cluster from a Amazon Elastic Kubernetes Service (EKS) deployment."
+lead: "Create and use MemoryDB for Redis using Amazon Elastic Kubernetes Service (EKS)."
+draft: false
+menu:
+  docs:
+    parent: "tutorials"
+weight: 42
```

Comment:
weight should be 44 to append it to existing tutorials

Reply:
Updated the document.